app/assets/javascripts/templates/common/table.hbs

```diff
@@ -0,0 +1,38 @@
+
+
+<div class="c-table">
+  <div class="table-header">
+    <div class="table-row">
+      {{#each headers}}
+        <div class="table-caption">{{ field_name }}</div>
+      {{/each}}
+    </div>
+  </div>
+  <div class="table-body">
```

Comment:
We should have used from the beginning the standard HTML table element. Reason: it seems we won't be able to make a table with flex because in order to compute the size of a cell, we need to know the size of the other ones (from the same column).